modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -1391,9 +1416,12 @@ createInferRequests(::ov::CompiledModel &compiled_model,
 
 // GOVExecutable implementation //////////////////////////////////////////////
 cv::gimpl::ov::GOVExecutable::GOVExecutable(const ade::Graph &g,
+                                            const cv::GCompileArgs &compileArgs,
                                             const std::vector<ade::NodeHandle> &nodes)
     : m_g(g), m_gm(m_g) {
 
+    m_inference_only =
+        cv::gapi::getCompileArg<cv::gapi::ov::inference_only>(compileArgs).has_value();
```

Comment:
probably you don't want to track all the individual fine-tuning knobs here, so I'd propose a configuration-like structure instead..

Reply:
Do you mean something like this?
```
namespace cv { namespace gapi { namespace wip { namespace { ov {
struct execution_config {
    bool enable_benchmark_mode = false;
};

} // ov
} // wip
} // gapi
} // cv

// Example:

auto compile_args = cv::compile_args(cv::gapi::wip::ov::execution_config{ true /* benchmark mode */ };
```

Reply:
Not at the API level. Inside.

Reply:
Done